profile/README.md

```diff
@@ -3,4 +3,4 @@
 - <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/studio.svg"/> Iterative Studio (Collaboration for ML Projects) | [website](https://studio.iterative.ai)
 - DVC Live (DVC Metrics Logger) | [Python API](https://github.com/iterative/dvclive) | [docs](https://dvc.org/doc/dvclive)
 - TPI: Terraform Provider Iterative | [code](https://github.com/iterative/terraform-provider-iterative) | [website & docs](https://registry.terraform.io/providers/iterative/iterative/latest/docs) | [Python API](https://github.com/iterative/tpi)
-- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/enterprise.svg"/> [awesome-iterative-projects](https://github.com/iterative/awesome-iterative-projects) | [example repos](https://github.com/iterative?q=topic%3Aexample&type=public&language=&sort=stargazers#org-repositories)
+- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/enterprise.svg"/> [awesome-iterative-projects](https://github.com/iterative/awesome-iterative-projects) | [example repos](https://github.com/iterative?q=topic%3Aexample&type=public&language=&sort=stargazers#org-repositories) | [Free course & certification](https://learn.iterative.ai)
```

Comment:
> `Free course & certification`

should wording be updated?

/CC @iterative/cse @jendefig @flippedcoder

Reply:
It looks like we won't have a second course until end of Q3 at the earliest.  I think we can update when we do.  I feel like training implies more 1:1 interaction.

Reply:
Also, thank you @casperdcl 🙏